src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -144,75 +147,62 @@ def export(
                         resp.content.decode("utf-8")))
             except Exception as ex:
                 self.__logger.warning("Failed to export metrics: %s", ex)
-                return MetricsExportResult.FAILURE
+                return MetricExportResult.FAILURE
 
-        return MetricsExportResult.SUCCESS
+        return MetricExportResult.SUCCESS
 
-    def _to_dynatrace_metric(self, metric: MetricRecord):
+    def _to_dynatrace_metric(self, metric: Metric):
         try:
-            attrs = dict(metric.labels)
-            if isinstance(metric.aggregator, aggregate.SumAggregator):
-                if not self._is_delta_export:
-                    self.__logger.info(
-                        "Received cumulative value which is currently"
-                        " not supported, using gauge instead.")
-                    # TODO: implement and use a Cumulative-to-Delta converter
+            attrs = dict(metric.attributes)
+            if isinstance(metric.point, Sum):
+                if isinstance(metric.point.value, float):
```

Comment:
You could just check if its an int and `else` assume float. I don't know how expensive it is to check this in Python, but I guess we would be doing this a lot

Reply:
It might still be a good idea to check it, though. Since types are not enforced, there could *technically* be anything in there . :thinking:

Reply:
Okay, lets leave it as is then. I realized that most of the time you will probably have floats anyway and therefore go down the first branch anyway.